src/write_gcm.c

```diff
@@ -3163,7 +3163,7 @@ static int wg_json_CreateTimeSeriesRequest(
     const wg_payload_t *head, const wg_payload_t **new_head, char **json)
 {
   char name[256];
-  int result = snprintf(name, sizeof(name), "project/%s",
+  int result = snprintf(name, sizeof(name), "projects/%s",
```

Comment:
This variable is unused. Should we just remove it?

Reply:
Done.